assets/css/navbar.css

```diff
@@ -92,3 +92,16 @@ html[dir='rtl'] .apertiumLogo {
 .nav-pills { /* sass-lint:disable-line class-name-format */
     cursor: pointer;
 }
+
+@media (min-width: 768px) {
+    .navbarMain {
+        align-items: flex-start;
+        display: flex;
```

Comment:
I'm afraid using flexbox is not currently an option for html-tools due to compatibility desires. After we switch to Bootstrap 4, then it'll be ok!

Reply:
I can add compatibility for all the browsers. Will it be okay then? If not where does flexbox fail?

Reply:
I'm not sure what you're saying. The browsers that html-tools supports right now don't support flexbox _at all_. So, a solution to this issue requires no usage of flexbox.

Reply:
Alright then. Should I work on coding a new solution or I should leave this as a solution for future?

Reply:
Up to you! Feel free to leave it open.